draft-dns-zone-digest.xml

```diff
@@ -515,14 +516,20 @@ RR(i) = owner | type | class | TTL | RDATA length | RDATA
             The zone digest is calculated using the algorithm
             described in <xref target="calculating-calculating"/>.
             Note in particular that the Digest field of any &RRNAME;
-            RRs MUST be zeroed prior to calculation of the digest.
+            RRs MUST be zeroed prior to calculation of the digest and
+            the RRSIGs for it must be removed.
```

Comment:
I avoiding saying that the RRSIG should be removed.  Instead I said it should be excluded from the digest calculation (section "Inclusion/Exclusion Rules").  My reasoning is that we want the DNSSEC signer to have all the right things in place, including NSEC, just before the digest is calculated.  Removing and then adding back the RRSIG seems more complex to me than just updating its RDATA.

Reply:
That makes sense.  I just wanted to specifically mention that ZONEMD RRSIGs aren't included in the hash.